Sources/VLCActionSheet/VLCActionSheetCell.swift

```diff
@@ -25,6 +27,7 @@ class VLCActionSheetCell: UICollectionViewCell {
         name.textColor = PresentationTheme.current.colors.cellTextColor
         name.font = UIFont.systemFont(ofSize: 15)
         name.translatesAutoresizingMaskIntoConstraints = false
+        name.setContentHuggingPriority(.defaultLow, for: .horizontal)
```

Comment:
Do we need to set `ContentHuggingPriority` ?

Reply:
This is to extend the name label all the way to the right so the checkmark can be at the rightmost edge. It can be moved to the subclass if it's breaking things.